src/pages/ServerRoleSummary.js

```diff
@@ -28,10 +28,6 @@ class ServerRoleSummary extends BaseWizardPage {
     super(props);
 
     this.checkInputs = ['nic-mapping', 'server-group'];
-    this.state = {
-      autoServers: undefined,
-      manualServers: undefined
-    };
```

Comment:
these are still referenced, is this just cleanup since they'll default to undefined?

Reply:
this is just clean up.